error.go

```diff
@@ -0,0 +1,5 @@
+package tprstorage
+
+import "github.com/giantswarm/microstorage"
+
+var invalidConfigError = microstorage.InvalidConfigError
```

Comment:
Oh, I made it inconsistent. I should also put here:

```
var notFoundError = microstorage.NotFoundError
```

This is because we want to use microstorage matchers like `microstorage.IsNotFoud`. This is important since since the using code may not know (and shouldn't rely on) underlying `Storage` implementation.